service/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.13.5 as bd
+WORKDIR /github.com/layer5io/sample-app-service
+ADD . .
+RUN GOPROXY=direct GOSUMDB=off go build -a -o /main .
+CMD ["/main"]
```

Comment:
Should we compile it to ./main instead of root directory? Thoughts @kanishkarj ?

Reply:
Oh yes, that would be better. I'll made that change!